===============================================================================

This is a patch for Open vSwitch.

This patch disables dependency computation for the generated Debian packages,
preventing improper dependencies from being detected when Open vSwitch is being
built for a target architecture that is different from the build host's.
Dependencies should appropriately being dealt with by other netkit-setup
scripts that are executed at a later stage of the filesystem image generation,
when openvswitch Debian packages are actually installed.

Applies successfully to commit bc651c85497d7a8c32e83b44986dae27dec3bcd5

===============================================================================

     Copyright (C) 2015  Massimo Rimondini (rimondini@ing.uniroma3.it)
     Computer Networks Research Group, Roma Tre University

     This program is free software: you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
     the Free Software Foundation, version 3 of the License.

     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.

     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.

=================================================================================
diff -Naurx .git ovs/debian/rules ovs-patched/debian/rules
--- ovs/debian/rules	2015-07-04 12:48:24.000000000 +0200
+++ ovs-patched/debian/rules	2015-07-04 12:43:41.000000000 +0200
@@ -138,7 +138,7 @@
 	dh_perl
 	dh_makeshlibs
 	dh_installdeb
-	dh_shlibdeps
+#	dh_shlibdeps
 	dh_gencontrol
 	dh_md5sums
 	dh_builddeb
